.travis.yml

```diff
@@ -54,10 +54,9 @@ install:
 
   # install TensorFlow (CPU version).
   - pip install tensorflow==1.9 --progress-bar off
-  
+
   # install cntk
   - if [[ "$KERAS_BACKEND" == "cntk" ]] || [[ "$TEST_MODE" == "PEP8_DOC" ]]; then
-      ./.travis/install_cntk.sh;
```

Comment:
What do you have against CNTK? :p

Reply:
That is my mistake... I'm sorry about that.

Reply:
No problem :)